Markdown/Markdown.sublime-syntax

```diff
@@ -2366,14 +2366,22 @@ contexts:
   table-cell-content:
     - match: (?={{balanced_emphasis}})
       push: table-cell-emphasis
-    - match: (?!{{backticks}})`+
-      scope: invalid.deprecated.unescaped-backticks.markdown
+    - include: table-cell-code-spans
     - include: table-cell-separators
     - include: images
-    - include: literals
+    - include: critics
+    - include: math-inline
+    - include: escapes
     - include: links
     - include: markups
 
+  table-cell-code-spans:
+    - match: (`+)[^`|]+(\1)
```

Comment:
Do we want to consider avoiding the use of Oniguruma by hard-coding the regex pattern for a single-backtick, double-backtick, triple-backtick etc. up to something reasonable like 5 or so?

Reply:
Normal code-spans even use branching in combination with `\1` and are even way slower than this implementation for table cells.

A benchmark with `\1` vs. fixed patterns using following table with 10k lines reveals a defference of 62ms vs. 40ms, while using default branching approach would use 95ms on my box.

```
| column 1 | column 2
| ---      | ---
| ` ` | ` me ` | `` ` `` | ` `` ` | ``foo`bar``
```

But well, maybe 5-6 backticks are enough in real world use cases in favor of a bit better performance, even though I doubt so many table cells with code spans are common.

Reply:
Does `` `{1,5}`` not work?

Reply:
The problem is that for balanced backticks, you'd still need to use a back reference for the closing backticks if you used ``(`{1,5})`` for the opening backticks

Reply:
Ah, okay. I misunderstood the difference as an unbounded length on the back red, rather than the use of one at all. Thanks!